prowler/providers/github/services/repository/repository_service.py

```diff
@@ -11,29 +11,44 @@ def __init__(self, provider):
         super().__init__(__class__.__name__, provider)
         self.repositories = self._list_repositories()
 
+    def _file_exists(self, repo, filename):
+        """Check if a file exists in the repository. Returns True if exists, False if not, None if error."""
+        try:
+            return repo.get_contents(filename) is not None
+        except Exception as error:
+            if "404" in str(error):
+                return False
+            else:
+                logger.error(
+                    f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                )
+                return None
+
     def _list_repositories(self):
         logger.info("Repository - Listing Repositories...")
         repos = {}
         try:
             for client in self.clients:
                 for repo in client.get_user().get_repos():
                     default_branch = repo.default_branch
+                    securitymd_exists = self._file_exists(repo, "SECURITY.md")
+                    # CODEOWNERS file can be in .github/, root, or docs/
+                    # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location
+                    codeowners_paths = [
+                        ".github/CODEOWNERS",
+                        "CODEOWNERS",
+                        "docs/CODEOWNERS",
+                    ]
+                    codeowners_exists = False
+                    for path in codeowners_paths:
+                        if self._file_exists(repo, path):
+                            codeowners_exists = True
+                            break
```

Comment:
We are not covering when `self._file_exists is None`.

Reply:
`None` is considered `false`, and `codeowners_exists` default value is `False` so it will keep it false.

Reply:
But remember that we need to keep the `None` value so the check does not run.

Reply:
You're right! No info, no finding!